profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/functions_to_wrap.c

```diff
@@ -302,7 +291,7 @@ void initLibrary(void)
         if (enableMiniDump != NULL && enableMiniDump[0] == '1')
         {
             // If DOTNET_DbgEnableMiniDump is set, the crash handler should call createdump when done
-            char* passthrough = getenv("DD_TRACE_CRASH_HANDLER_PASSTHROUGH");
+            char* passthrough = getenv("DD_INTERNAL_CRASHTRACKING_PASSTHROUGH");
```

Comment:
Use constants for env var names to avoid repeating them and maybe introduce typos

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5898/commits/904b7eb2a9d73f42be6febfb5d38ebde70b31878